### PR TITLE
Carthage Compatibility

### PR DESCRIPTION
**Pull request by adly-holler**
[Carthage](http://github.com/Carthage/Carthage) is a decentralized package manager for Swift made by some folks from GitHub. It's way less invasive than CocoaPods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,20 +17,7 @@ Das Quadrat is Foursquare API wrapper written in Swift.
...
-2. Drag-and-drop `Quadrat.xcodeproj` into your project. The project has two targets: Quadrat.framework for OSX project, QuadratTouch.framework for iOS projects. 
-3. Add new target in "Build Phases" -> "Target Dependencies".
-4. Click the `+` button at the top left of the panel and choose "New copy files phase".
-  * Rename the new phase to "Copy Frameworks".
-  * Set the "Destination" to "Frameworks".
-5. Add Quadrat framework to this phase.
-6. Here is the [image](https://cloud.githubusercontent.com/assets/239692/5367193/367f8640-7ffa-11e4-8b9b-88cef33bcd79.png) to visualise the result.
-7. Read [wiki](https://github.com/Constantine-Fry/das-quadrat/wiki) for details. 
-
-You can add source code files directly into your project to support iOS 7.0.
```
Comment (Constantine-Fry) [inline README.md]: Well, not everyone works with carthage, someone will need to install it manually. Keep this description, please.

**Comment by adcooley** [comment]
@Constantine-Fry added back the removed README section. Would be great to get this merged in!

**Comment by Constantine-Fry** [comment]
Hi, guys. Thank you for your PRs. Did you notice that you made two identical PR? Which one would you like the merge?

File: QuadratTouch.podspec [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  s.name         = "QuadratTouch"
+  s.version      = "1.0.0"
+  s.summary      = "Das Quadrat is Foursquare API wrapper written in Swift."
+
+  s.description  = <<-DESC
+                   Das Quadrat is Foursquare API wrapper written in Swift.
+                   DESC
+
+  s.homepage     = "https://github.com/Tripstr/das-quadrat"
```
Comment (Constantine-Fry) [inline QuadratTouch.podspec]: Let's change it to the original repository.

File: QuadratTouch.podspec [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  s.ios.deployment_target = "8.0"
+  s.source       = { :git => "https://github.com/Tripstr/das-quadrat.git", :branch => "master" }
```
Comment (Constantine-Fry) [inline QuadratTouch.podspec]: And here too.

**Pull request merged**